worker/tasks.py

```diff
@@ -393,32 +393,25 @@ def test(ctx):
             shell=SHELL
         );
 
+    mediasoup_worker_test = 'mediasoup-worker-test.exe' if os.name == 'nt' else 'mediasoup-worker-test';
```

Comment:
Looks like this refactoring made code harder to read. What is the benefit of extracting the name into a variable if the rest of the executed command is platform-specific anyway?

Reply:
Because we may want to change command line arguments given to the test binary and I prefer to not have all them duplicated. There is no platform-specific stuff here other than the binary name.

Reply:
What I'm saying is that both command and arguments are different between Windows and macOS/Windows. So in terms of code complexity you have one `if` on this line and one more `if` below, which leads to 4 potential branches, while only 2 of them are valid.

You could just inline `mediasoup-worker-text[.exe]` into the commands below like it was before and it would make code easier to read.

This variable doesn't bring any benefit, I think the old version was better.

Reply:
Then I don't understand what you mean. Old version had duplicate code:

![CleanShot-2024-01-01-at-14 04 44@2x](https://github.com/versatica/mediasoup/assets/16191/6ec22012-6da9-4ccf-9e15-2e30c3d509c4)

New version doesn't have duplicate code:

<img width="898" alt="CleanShot-2024-01-01-at-14 05 45@2x" src="https://github.com/versatica/mediasoup/assets/16191/1d932548-ba5b-4d03-90dd-59ca3277bce5">

Reply:
That is exactly what I mean: old code had duplicate and having a variable might potentially make sense, but now even commands are different.

Why is it better to write `{mediasoup_worker_test}` instead of `mediasou-worker-test`?

Reply:
> but now even commands are different.

What? where? 😀

There is a `lcov` command that only runs on Linux/macOS and it was there since day 0. Other than that we have a single call to `mediasoup-worker-test[.exe]` with **same** arguments.

> Why is it better to write `{mediasoup_worker_test}` instead of `mediasou-worker-test`?

Because in Windows it must be `mediasou-worker-test.exe`.

Reply:
Okay, now I see. It is not just refactoring, the code is also different.

Now makes sense, next time try to create two commits: one with refactoring, second that introduces new things. That way it is easier to review.